content/blog/platypus-platform-pulumi-at-1000-nodes/index.md

```diff
@@ -0,0 +1,119 @@
+
+{{< figure src="https://www.pulumi.com/uploads/pulumi-insights-search.gif" caption="Figure: Search for any resource with Pulumi Insights">}}
+
+### Security and Compliance
+
+In the modern parlance, when you say DevOps, you mean DevSecOps. Pulumi is designed to be secure by default. Pulumi Cloud offers full [role-based access control (RBAC) functionality](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/) including deep integration with [GitHub teams](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/#github-based-teams) and [SAML-based SSO](https://www.pulumi.com/docs/pulumi-cloud/access-management/saml/), managed secrets and flexibly-defined secure environments with [Pulumi ESC](https://www.pulumi.com/product/esc/), and policy-as-code provided by [Pulumi Crossguard](https://www.pulumi.com/crossguard/). Most importantly all of these features are deeply integrated across the platform, creating an air-tight system with all the guardrails you need for managing security and access.
+
+### Auditability
+
+Every action a user takes in Pulumi can be tracked via the [audit log](https://www.pulumi.com/docs/pulumi-cloud/audit-logs/) which is searchable in two clicks from the Pulumi Cloud homepage dashboard. Audit logs can be filtered by user with one more click. Creating automated backups of your audit logs is a [first-class feature](https://www.pulumi.com/docs/pulumi-cloud/audit-logs/#automated-export). You will never have to worry about responding quickly when someone asks about an event that happened in your system. Also, each deployment and update has logs directly visible from the Pulumi Cloud app, regardless of how it was initiated.
+
+{{< figure src="/images/docs/guides/self-hosted/auditlogs.png" caption="Figure: Viewing the audit log in Pulumi Cloud">}}
+
+### Developer Experience
+
+Probably the most compelling aspect of Pulumi is the developer experience. [Developers love Pulumi](https://www.pulumi.com/testimonials/), because they get to use their preferred tools. General purpose programming languages, visual IDEs, command-line tools, and products with an API-driven architecture are what developers want, and it's what Pulumi delivers in spades.
+
+With Pulumi templates and custom internal component resources in place, developers can drive their own DevOps, provisioning their own infrastructure resources and managing their own deployments directly, reducing bottlenecks in platform teams. Product engineering teams can self-service with a stream-lined workflow that stays compliant with company policy by default. Deep in the code of their favorite programming languages, your developers will never even know they are following the company rules.
+
+{{< figure src="pulumi-ide.png" caption="Figure: Using C# to write a Pulumi program in VS Code">}}
+
+### More to Come
```

Comment:
so Engin's post makes reference at the end to your post. I believe his comes out first. So it would be good to cross-link to his post and maybe acknowledge the intro he makes into your post

https://github.com/pulumi/docs/pull/12296

Reply:
added that in w/ link that should exist once Engin's post is out